install-and-configure/advanced-configuration/eks-optimized/amp-with-remote-write.md

```diff
@@ -0,0 +1,200 @@
+# AMP with Kubecost Prometheus (remote_write)
+
+## See also
+
+* [AMP Overview](/install-and-configure/advanced-configuration/eks-optimized/aws-amp-integration.md)
+* [AWS Agentless AMP](/install-and-configure/advanced-configuration/eks-optimized/kubecost-agentless-amp.md)
+* [AWS Distro for Open Telemetry](/install-and-configure/advanced-configuration/eks-optimized/kubecost-aws-distro-open-telemetry.md)
+
+## Overview
+
+When the Amazon Managed Service for Prometheus integration is enabled, the bundled Prometheus server in the Kubecost Helm Chart is configured in the remote\_write mode. The bundled Prometheus server sends the collected metrics to Amazon Managed Service for Prometheus using the AWS SigV4 signing process. All metrics and data are stored in Amazon Managed Service for Prometheus, and Kubecost queries the metrics directly from Amazon Managed Service for Prometheus instead of the bundled Prometheus. It helps customers not worry about maintaining and scaling the local Prometheus instance.
+
+Kubecost has multiple methods for multi-cluster. There may be performance limits to how many clusters/nodes can be supported on a single AMP instance. Please contact Kubecost support for more information.
+
```

Comment:
I've rewritten this section, but need confirmation the user must start with these, rather than these being details of completion of the tutorial

Reply:
looks good to me. This doc is relatively old. Should still be relevant, but not worth a ton of time.